daemon/api.go

```diff
@@ -2733,6 +2754,7 @@ func postApps(c *Command, r *http.Request, user *auth.UserState) Response {
 	if err := decoder.Decode(&inst); err != nil {
 		return BadRequest("cannot decode request body into service operation: %v", err)
 	}
+	// XXX: decoder.More()
```

Comment:
This should not happen unless there's mulitple `servicestate.Instruction` being sent in the body. Also, we're only intersted in one full `Instruction` object, which is what a single call to `Decode()` should provide.